project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/OpenCodeMetricsViewHandler.java

```diff
@@ -0,0 +1,87 @@
+ *   list of conditions and the following disclaimer.
+ * 
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+package com.samsung.sec.dexter.eclipse.ui;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.core.commands.IHandler;
+import org.eclipse.ui.PartInitException;
+import org.eclipse.ui.PlatformUI;
+
+import com.google.common.base.Strings;
+import com.samsung.sec.dexter.core.exception.DexterRuntimeException;
+import com.samsung.sec.dexter.core.util.DexterClient;
+import com.samsung.sec.dexter.core.util.DexterUtil;
+import com.samsung.sec.dexter.eclipse.ui.util.EclipseUtil;
+
+public class OpenCodeMetricsViewHandler extends AbstractHandler implements IHandler {
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException {
+		try {
+			final String url = DexterClient.getInstance().getDexterCodeMetricsUrl();			
+			if(DexterUtil.getOS() == DexterUtil.OS.WINDOWS){
+				String exePath = DexterUtil.readRegistry("HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\firefox.exe", "Path");
+				
+				if(!Strings.isNullOrEmpty(exePath)){
+					Runtime.getRuntime().exec("\"" + exePath + "\\firefox.exe\" " + url);
+					return null;
+				}
+				
+				exePath = DexterUtil.readRegistry("HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\chrome.exe", "Path");
+				
+				if(!Strings.isNullOrEmpty(exePath)){
+					Runtime.getRuntime().exec("\"" + exePath + "\\chrome.exe\" " + url);
+					return null;
+				}
+				
+				PlatformUI.getWorkbench().getBrowserSupport().getExternalBrowser().openURL(new URL(url));
+			} else {
+				PlatformUI.getWorkbench().getBrowserSupport().getExternalBrowser().openURL(new URL(url));
+			}
+        } catch (PartInitException e) {
```

Comment:
If exception handler is same, you can use like below:

catch(PartInitException | MalformedURLException | IOException e){
    handleCommonError(e);
}

read this: http://docs.oracle.com/javase/7/docs/technotes/guides/language/catch-multiple.html